skbio/stats/ordination/_ordination_results.py

```diff
@@ -251,6 +248,8 @@ def plot(self, df=None, column=None, axes=(0, 1, 2), axis_labels=None,
 
         # derived from
         # http://matplotlib.org/examples/mplot3d/scatter3d_demo.html
+        import matplotlib.pyplot as plt
+        from mpl_toolkits.mplot3d import Axes3D
```

Comment:
Travis-CI is failing when flake8 runs. Add `  # noqa` to the end of this line to fix that (note that flake8 requires two spaces before `#` for inline comments). To test this out locally you can run `flake8 skbio`.